.gitignore

```diff
@@ -1,20 +1,21 @@
-node_modules
-npm-debug.log
-yarn.lock
-*data.json
-*data*
-*.db
-.idea/
-*.bak
-*.new
-*.log
-v8.json
-*.DS_store
-isolate*.log
-.esm-cache
-.sessionStorage
-.localStorage
-/types/**/*.ts
-!/types/**/*.d.ts
-/gun.ts
-/temp/
\ No newline at end of file
+node_modules
+npm-debug.log
+yarn.lock
+*data.json
+*data*
+*.db
+.idea/
+*.bak
+*.new
+*.log
+v8.json
+*.DS_store
+isolate*.log
+.esm-cache
+.sessionStorage
+.localStorage
+/types/**/*.ts
+!/types/**/*.d.ts
+/gun.ts
+/temp/
+.nyc_output/
```

Comment:
This btw is a prefect example of why you need linting. This file clearly had CRLF line breaks, and because I added 1 small change at the end my editor detected inconsistencies and normalized it. The `.eslintignore` file actually has a list of such files in the *.js realm which fail to be consistent with the rest of the project.